test/net/sourceforge/kolmafia/request/PlaceRequestTest.java

```diff
@@ -74,7 +79,75 @@ void itShouldSetTheToolbeltAsAFreePullInTTT() {
     assertEquals(
         -1, freePulls.indexOf(toolbelt), "toolbelt should not be in Free Pulls after TTT fades");
   }
-}
 
-// Generated with love by TestMe :) Please report issues and submit feature requests at:
-// http://weirddev.com/forum#!/testme
+  @Nested
+  class Speakeasy {
+    @BeforeEach
+    public void initializeCharPrefs() {
+      KoLCharacter.reset("PlaceRequestTestFakeSotVisit");
+      KoLCharacter.reset(true);
+    }
+
+    @AfterEach
+    public void resetCharAndPrefs() {
+      KoLCharacter.reset("");
+      KoLCharacter.reset(true);
+      KoLCharacter.setUserId(0);
+    }
+
+    @Test
+    public void itShouldGetParcelLocationFromFirstVisit() {
+      String prefName = "_sotParcelLocation";
+      assertEquals("", Preferences.getString(prefName), "Preference already set.");
```

Comment:
What? Why are these tests not using withProperty and ensuring the property is set the way you expect, rather than guarding against leaks from other tests? You can't count on the order tests are run. They must be independent.

Reply:
This wasn't a deliberate guard.  I thought of this test as testing a change in state and my generic method has been to check the old state, do something, check the state and confirm that it changed.  That said, I think reworking to use withProperty is a better way.